config.go

```diff
@@ -76,12 +90,33 @@ func GenerateConfig() Config {
 }
 
 func loadConfig(configFile string) Config {
+	// Default config
+	config := Config{
+		Bind:       "0.0.0.0:3000",
+		ServePath:  "/i/",
+		UploadPath: "./uploads/",
+	}
```

Comment:
Do we wanna just write this once in here since used twice?

Reply:
Deal. I added a function that returns the default config.